lib/daru/date_time/index.rb

```diff
@@ -226,7 +226,7 @@ def each(&block)
       to_a.each(&block)
     end
 
-    attr_reader :frequency, :offset, :periods
+    attr_reader :frequency, :offset, :periods, :keys
```

Comment:
Without this change you can't call `vector.at(*[1,2])` (or as far as I can tell, any `.at`) on a vector with a DateTime index (bug #331). This is because DateTime index inherits from index which calls `keys (position)` which has `@keys[value]`.

Without this change the DateTime indexed vector has no @keys.